src/components/MediaCard.jsx

```diff
@@ -0,0 +1,8 @@
+import Media from './Media'
+
+function MediaCard(props) {
+  const info = props.info
+  return (<Media info={info} />);
+}
```

Comment:
it's not clear why do you need the `MediaCard` component if all it does is render the `Media` component. 
The initial idea was that the `MediaCard` will handle the images (including the title and description), and `Media` will just handle the image itself (whether it's a picture or video)